pkg/admission/conf/am_conf.go

```diff
@@ -118,7 +118,10 @@ func NewAdmissionControllerConf(configMaps []*v1.ConfigMap) *AdmissionController
 }
 
 func (acc *AdmissionControllerConf) RegisterHandlers(configMaps informersv1.ConfigMapInformer) {
-	configMaps.Informer().AddEventHandler(&configMapUpdateHandler{conf: acc})
+	_, err := configMaps.Informer().AddEventHandler(&configMapUpdateHandler{conf: acc})
```

Comment:
It seems to me this error should be viewed as fatal, and we should propagate it to stop k8shime.